fractions/keycloak-microprofile-jwt/README.adoc

```diff
@@ -1,9 +1,7 @@
-= Keycloak MicroProfile JWT
+= Keycloak Eclipse MicroProfile JWT
```

Comment:
Not here. The fraction is called `keycloak-microprofile-jwt`, which is already quite a mouthful :-)

Reply:
And the full name is used in the 1st sentence below. So we should be able to omit the "Eclipse" word here?

Reply:
Yes, since the name of the fraction is _Keycloak MicroProfile JWT_, I think we can omit _Eclipse_ from the title.
```suggestion
= Keycloak MicroProfile JWT
```